src/web/components/Button/index.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Button({
+  children,
+  onClick,
+  color,
+  disabled,
+  size,
+  outlined,
+  inverted,
+  rounded,
+  loading,
+}) {
+  const colorClass = color && ` is-${color}`;
+  const sizeClass = size && ` is-${size}`;
+  const outlinedClass = outlined ? ' is-outlined' : '';
```

Comment:
Why don't you use the same syntax as in line 15 - 16 for others classes ? 🤔

Reply:
When the variable is boolean  is not set it's return the string "false" and not an empty string